<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWidget</class>
 <widget class="QWidget" name="ServerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(0, 0, 0);
}
QPushButton{
background-color: rgb(255, 170, 0);
}

QTabWidget{
background-color: rgb(85, 170, 0);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_9">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">QWidget{
	font: 12pt &quot;Arial&quot;;
background-color: rgb(255, 255, 255);
	background-color: rgb(85, 255, 255);
}</string>
             </property>
             <property name="text">
              <string>Show window</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEditRead">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 255);
font: 14pt &quot;Arial&quot;;</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">QWidget{
	font: 12pt &quot;Arial&quot;;
background-color: rgb(85, 255, 255);
}</string>
             </property>
             <property name="text">
              <string>Write  window</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textEditWrite">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 255, 255);
font: 14pt &quot;Arial&quot;;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btnSend">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>send</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>288</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnClose">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 255, 255);</string>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="student">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
          <widget class="QTableWidget" name="tableWidgetStu">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>301</width>
             <height>421</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 11pt &quot;Arial&quot;;
font: 75 12pt &quot;Arial&quot;;
background-color: rgb(0, 255, 255);</string>
           </property>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>2</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>3</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>4</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>5</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>6</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>7</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>8</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>9</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>姓名</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>年龄</string>
            </property>
           </column>
          </widget>
         </widget>
         <widget class="QWidget" name="programmer">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
          <widget class="QTableWidget" name="tableWidgetPro">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>291</width>
             <height>401</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 255, 255);
font: 75 12pt &quot;Arial&quot;;</string>
           </property>
           <row>
            <property name="text">
             <string>1</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>2</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>3</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>4</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>5</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>姓名</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>年龄</string>
            </property>
           </column>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnToSql">
           <property name="minimumSize">
            <size>
             <width>56</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">alternate-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));
font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>add to sql</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSqlOut">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>sql to out</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QPushButton" name="btnFirst">
           <property name="styleSheet">
            <string notr="true">font: 11pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>first</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnTurnUp">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>turn up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnTurnDown">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>turn down</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnEnd">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Arial&quot;;</string>
           </property>
           <property name="text">
            <string>end</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 255, 255);
font: 12pt &quot;Arial&quot;;</string>
         </property>
         <property name="title">
          <string>MySql  Connecttion</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>LoHost</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditLocaHost"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Datebs</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditDateBs"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>userNa</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditUseName"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="styleSheet">
                   <string notr="true">font: 11pt &quot;Arial&quot;;
font: 12pt &quot;Arial&quot;;</string>
                  </property>
                  <property name="text">
                   <string>Passwd</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditPassWd"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>28</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="styleSheet">
               <string notr="true">QWidget{background-color: rgb(85, 255, 0);}
</string>
              </property>
              <property name="text">
               <string>Connect</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
